cmd/environment.go

```diff
@@ -18,9 +19,23 @@ func (h *Handler) Environment(ctx context.Context, req *entity.CommandRequest) e
 		return err
 	}
 
-	environment, err := ui.PromptEnvironments(project.Environments)
-	if err != nil {
-		return err
+	var environment *entity.Environment
+	if len(req.Args) > 0 {
+		// Create new environment
+		environment, err = h.ctrl.CreateEnvironment(ctx, &entity.CreateEnvironmentRequest{
```

Comment:
This doesn't seem to create an ephemeral environment (like the PR indicates) and rather just creates a normal environment.

Reply:
Ya sorry the titles a misnomer